### DISCUSSION TITLE
Batch operations don't work on Synology hardware

### DISCUSSION THREAD

**Discussion by djatmdfhr** [Difegue/LANraragi#1019] [Q&A] [answered]
**LRR Version and OS**
Docker(on synology DSM 7.1.1-42692)
LRR v0.9.10(latest)

**Bug Details**
The metadata is not imported.
My file name seems to be appropriate. 
ex) {number} [artist] title.zip
![image](https://github.com/Difegue/LANraragi/assets/174906545/9e9e1224-209e-43d8-a3d2-77169e53f04b)
And this is the problem I'm having.
I set the plugin to hitomi and pressed the start task button, but the error continues. (on Hitomi.pm line 119?)

**Matching Logs**
![image](https://github.com/Difegue/LANraragi/assets/174906545/79b9d687-44cf-4026-96dd-3ce265398d41)

**Comment by Difegue** [comment, marked as answer]
The normal image does not work on Synology thanks to them using a very old LTS as the base for the OS, which causes issues with plugins and batch operations thanks to missing dependencies.  

Lanraragi v0.8.4 is the last normal release that works on Synology.  

If you need the latest versions, you can try building the [legacy Dockerfile.](https://github.com/Difegue/LANraragi/blob/dev/tools/build/docker/Dockerfile-legacy) 

Another user made a build of the latest version if that can help you out: https://hub.docker.com/r/bobthurgen/lanraragi-legacy-arm64  
(Unfortunately it's not worth it for me to push docker hub builds of the legacy file alongside the main one in CI..)

**Reply by Difegue** [reply]
Also if you are using a reverse proxy to access it, that breaks the batch operations page due to the reliance on websockets.  
Consider using the "host" option rather than the "bridge" option, mapping port 3000 to 3000.

**Reply by djatmdfhr** [reply]
Thx for replying.
The answer was to use the host option.
I've tested a lot of cases, including using v0.8.4.
Finally, when I used host option, I found that the batch operation works properly even in the latest version.
So my problem is solved now, thx very much.

**Reply by eao** [reply]
Do delete operations work normally for you?  If so, what version of the app are you on?

**Reply by djatmdfhr** [reply]
Yeah, it works well. I'm using v0.9.10, the latest version that is uploaded on synology docker.
But at first, I was forced to stop uploading because I didn't have enough stoarge, then there was a storage issues.
If you are in same situation, you have to delete temp files by ssh. (Because you can't access to the files on DSM)
Use commands like cd, du -sh to find and access to large files on LRR folder, then delete it with rm -r.

**Reply by KobayashiNori** [reply]
I'm in the same situation.
This is DSM 7.1.1-42962 Update 6, and I'm using LANarragi's 0.9.10.
The same error phrase occurs and cannot be resolved by using host and 0.8.4 version.
